diff --git a/config.def.h b/config.def.h
index fd09d72..f859a34 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#161b1d", /* color base00 */
+	"#d22d72", /* color base08 */
+	"#568c3b", /* color base0B */
+	"#8a8a0f", /* color base0A */
+	"#257fad", /* color base0D */
+	"#6b6bb8", /* color base0E */
+	"#2d8f6f", /* color base0C */
+	"#7ea2b4", /* color base05 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#5a7b8c", /* color base03 */
+	"#d22d72", /* color base08 */
+	"#568c3b", /* color base0B */
+	"#8a8a0f", /* color base0A */
+	"#257fad", /* color base0D */
+	"#6b6bb8", /* color base0E */
+	"#2d8f6f", /* color base0C */
+	"#ebf8ff", /* color base07 */
+
+	/* additional colors */
+	"#935c25", /* color base09 */
+	"#b72dd2", /* color base0F */
+	"#1f292e", /* color base01 */
+	"#516d7b", /* color base02 */
+	"#7195a8", /* color base04 */
+	"#c1e4f6", /* color base06 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#516d7b", /* color base02 */
+	"#ebf8ff", /* color base07 */
 };
 
 
