diff --git a/config.def.h b/config.def.h
index 64e75b8..69dfe8b 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,24 +62,32 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#000000", /* color   0 */
+	"#e92f2f", /* color   1 */
+	"#0ed839", /* color   2 */
+	"#dddd13", /* color   3 */
+	"#3b48e3", /* color   4 */
+	"#f996e2", /* color   5 */
+	"#23edda", /* color   6 */
+	"#ababab", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#343434", /* color   8 */
+	"#e92f2f", /* color   9 */
+	"#0ed839", /* color  10 */
+	"#dddd13", /* color  11 */
+	"#3b48e3", /* color  12 */
+	"#f996e2", /* color  13 */
+	"#23edda", /* color  14 */
+	"#f9f9f9", /* color  15 */
+
+	/* additional colors */
+	"#e09448", /* color  16 */
+	"#69542d", /* color  17 */
+	"#040404", /* color  18 */
+	"#102015", /* color  19 */
+	"#555555", /* color  20 */
+	"#e0e0e0", /* color  21 */
 
 	[255] = 0,
 
@@ -94,7 +102,7 @@ static const char *colorname[] = {
  */
 static unsigned int defaultfg = 7;
 static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+static unsigned int defaultcs = 7;
 
 /*
  * Colors used, when the specific fg == defaultfg. So in reverse mode this
