diff --git a/config.def.h b/config.def.h
index 64e75b8..7a24036 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,24 +62,32 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#201602", /* color   0 */
+	"#c35359", /* color   1 */
+	"#18974e", /* color   2 */
+	"#a88339", /* color   3 */
+	"#477ca1", /* color   4 */
+	"#8868b3", /* color   5 */
+	"#75a738", /* color   6 */
+	"#948e48", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#6c6823", /* color   8 */
+	"#b36144", /* color   9 */
+	"#302e00", /* color  10 */
+	"#5f5b17", /* color  11 */
+	"#86813b", /* color  12 */
+	"#ccc37a", /* color  13 */
+	"#b3588e", /* color  14 */
+	"#faf0a5", /* color  15 */
+
+	/* additional colors */
+	"#b36144", /* color  16 */
+	"#b3588e", /* color  17 */
+	"#302e00", /* color  18 */
+	"#5f5b17", /* color  19 */
+	"#86813b", /* color  20 */
+	"#ccc37a", /* color  21 */
 
 	[255] = 0,
 
@@ -92,9 +100,9 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+static unsigned int defaultfg = 19;
+static unsigned int defaultbg = 15;
+static unsigned int defaultcs = 19;
 
 /*
  * Colors used, when the specific fg == defaultfg. So in reverse mode this
