diff --git a/config.def.h b/config.def.h
index fd09d72..c3c4646 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,30 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#2f1e2e", /* color   0 */
+	"#ef6155", /* color   1 */
+	"#48b685", /* color   2 */
+	"#fec418", /* color   3 */
+	"#06b6ef", /* color   4 */
+	"#815ba4", /* color   5 */
+	"#5bc4bf", /* color   6 */
+	"#a39e9b", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#776e71", /* color   8 */
+	"#f99b15", /* color   9 */
+	"#41323f", /* color  10 */
+	"#4f424c", /* color  11 */
+	"#8d8687", /* color  12 */
+	"#b9b6b0", /* color  13 */
+	"#e96ba8", /* color  14 */
+	"#e7e9db", /* color  15 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#a39e9b",
+	"#2f1e2e",
 };
 
 
