diff --git a/config.def.h b/config.def.h
index fd09d72..65baaff 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#0b1c2c", /* color   0 */
+	"#bf8b56", /* color   1 */
+	"#56bf8b", /* color   2 */
+	"#8bbf56", /* color   3 */
+	"#8b56bf", /* color   4 */
+	"#bf568b", /* color   5 */
+	"#568bbf", /* color   6 */
+	"#cbd6e2", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#627e99", /* color   8 */
+	"#bf8b56", /* color   9 */
+	"#56bf8b", /* color  10 */
+	"#8bbf56", /* color  11 */
+	"#8b56bf", /* color  12 */
+	"#bf568b", /* color  13 */
+	"#568bbf", /* color  14 */
+	"#f7f9fb", /* color  15 */
+
+	/* additional colors */
+	"#bfbf56", /* color  16 */
+	"#bf5656", /* color  17 */
+	"#223b54", /* color  18 */
+	"#405c79", /* color  19 */
+	"#aabcce", /* color  20 */
+	"#e5ebf1", /* color  21 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#cbd6e2",
+	"#0b1c2c",
 };
 
 
