diff --git a/config.def.h b/config.def.h
index 64e75b8..ca76df4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,24 +62,32 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#1d1f21", /* color   0 */
+	"#CC342B", /* color   1 */
+	"#198844", /* color   2 */
+	"#FBA922", /* color   3 */
+	"#3971ED", /* color   4 */
+	"#A36AC7", /* color   5 */
+	"#3971ED", /* color   6 */
+	"#c5c8c6", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#969896", /* color   8 */
+	"#CC342B", /* color   9 */
+	"#198844", /* color  10 */
+	"#FBA922", /* color  11 */
+	"#3971ED", /* color  12 */
+	"#A36AC7", /* color  13 */
+	"#3971ED", /* color  14 */
+	"#ffffff", /* color  15 */
+
+	/* additional colors */
+	"#F96A38", /* color  16 */
+	"#3971ED", /* color  17 */
+	"#282a2e", /* color  18 */
+	"#373b41", /* color  19 */
+	"#b4b7b4", /* color  20 */
+	"#e0e0e0", /* color  21 */
 
 	[255] = 0,
 
@@ -92,9 +100,9 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+static unsigned int defaultfg = 19;
+static unsigned int defaultbg = 15;
+static unsigned int defaultcs = 19;
 
 /*
  * Colors used, when the specific fg == defaultfg. So in reverse mode this
