diff --git a/config.def.h b/config.def.h
index fd09d72..8e3ddd2 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#2b2b2b", /* color   0 */
+	"#da4939", /* color   1 */
+	"#a5c261", /* color   2 */
+	"#ffc66d", /* color   3 */
+	"#6d9cbe", /* color   4 */
+	"#b6b3eb", /* color   5 */
+	"#519f50", /* color   6 */
+	"#e6e1dc", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#5a647e", /* color   8 */
+	"#cc7833", /* color   9 */
+	"#272935", /* color  10 */
+	"#3a4055", /* color  11 */
+	"#d4cfc9", /* color  12 */
+	"#f4f1ed", /* color  13 */
+	"#bc9458", /* color  14 */
+	"#f9f7f3", /* color  15 */
+
+	/* additional colors */
+	"#cc7833", /* color  16 */
+	"#bc9458", /* color  17 */
+	"#272935", /* color  18 */
+	"#3a4055", /* color  19 */
+	"#d4cfc9", /* color  20 */
+	"#f4f1ed", /* color  21 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#e6e1dc",
+	"#2b2b2b",
 };
 
 
