diff --git a/config.def.h b/config.def.h
index 64e75b8..1132309 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,24 +62,24 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#111111", /* color   0 */
+	"#ff4136", /* color   1 */
+	"#2ecc40", /* color   2 */
+	"#ffdc00", /* color   3 */
+	"#0074d9", /* color   4 */
+	"#b10dc9", /* color   5 */
+	"#7fdbff", /* color   6 */
+	"#bbbbbb", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#777777", /* color   8 */
+	"#ff851b", /* color   9 */
+	"#333333", /* color  10 */
+	"#555555", /* color  11 */
+	"#999999", /* color  12 */
+	"#dddddd", /* color  13 */
+	"#85144b", /* color  14 */
+	"#ffffff", /* color  15 */
 
 	[255] = 0,
 
@@ -94,7 +94,7 @@ static const char *colorname[] = {
  */
 static unsigned int defaultfg = 7;
 static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+static unsigned int defaultcs = 7;
 
 /*
  * Colors used, when the specific fg == defaultfg. So in reverse mode this
