diff --git a/config.def.h b/config.def.h
index fd09d72..cb5f086 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#1b181b", /* color   0 */
+	"#ca402b", /* color   1 */
+	"#379a37", /* color   2 */
+	"#bb8a35", /* color   3 */
+	"#516aec", /* color   4 */
+	"#7b59c0", /* color   5 */
+	"#159393", /* color   6 */
+	"#ab9bab", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#776977", /* color   8 */
+	"#ca402b", /* color   9 */
+	"#379a37", /* color  10 */
+	"#bb8a35", /* color  11 */
+	"#516aec", /* color  12 */
+	"#7b59c0", /* color  13 */
+	"#159393", /* color  14 */
+	"#f7f3f7", /* color  15 */
+
+	/* additional colors */
+	"#a65926", /* color  16 */
+	"#cc33cc", /* color  17 */
+	"#292329", /* color  18 */
+	"#695d69", /* color  19 */
+	"#9e8f9e", /* color  20 */
+	"#d8cad8", /* color  21 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#ab9bab",
+	"#1b181b",
 };
 
 
