diff --git a/config.def.h b/config.def.h
index fd09d72..606d649 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#101010", /* color   0 */
+	"#7c7c7c", /* color   1 */
+	"#8e8e8e", /* color   2 */
+	"#a0a0a0", /* color   3 */
+	"#686868", /* color   4 */
+	"#747474", /* color   5 */
+	"#868686", /* color   6 */
+	"#b9b9b9", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#525252", /* color   8 */
+	"#999999", /* color   9 */
+	"#252525", /* color  10 */
+	"#464646", /* color  11 */
+	"#ababab", /* color  12 */
+	"#e3e3e3", /* color  13 */
+	"#5e5e5e", /* color  14 */
+	"#f7f7f7", /* color  15 */
+
+	/* additional colors */
+	"#999999", /* color  16 */
+	"#5e5e5e", /* color  17 */
+	"#252525", /* color  18 */
+	"#464646", /* color  19 */
+	"#ababab", /* color  20 */
+	"#e3e3e3", /* color  21 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#b9b9b9",
+	"#101010",
 };
 
 
