diff --git a/config.def.h b/config.def.h
index 64e75b8..d972289 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,24 +62,24 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#1b1918", /* color   0 */
+	"#f22c40", /* color   1 */
+	"#5ab738", /* color   2 */
+	"#d5911a", /* color   3 */
+	"#407ee7", /* color   4 */
+	"#6666ea", /* color   5 */
+	"#00ad9c", /* color   6 */
+	"#a8a19f", /* color   7 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#766e6b", /* color   8 */
+	"#df5320", /* color   9 */
+	"#2c2421", /* color  10 */
+	"#68615e", /* color  11 */
+	"#9c9491", /* color  12 */
+	"#e6e2e0", /* color  13 */
+	"#c33ff3", /* color  14 */
+	"#f1efee", /* color  15 */
 
 	[255] = 0,
 
@@ -94,7 +94,7 @@ static const char *colorname[] = {
  */
 static unsigned int defaultfg = 7;
 static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+static unsigned int defaultcs = 7;
 
 /*
  * Colors used, when the specific fg == defaultfg. So in reverse mode this
