diff --git a/config.def.h b/config.def.h
index fd09d72..51f13a4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#000000", /* color base00 */
+	"#ff0000", /* color base08 */
+	"#33ff00", /* color base0B */
+	"#ff0099", /* color base0A */
+	"#0066ff", /* color base0D */
+	"#cc00ff", /* color base0E */
+	"#00ffff", /* color base0C */
+	"#d0d0d0", /* color base05 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#808080", /* color base03 */
+	"#ff0000", /* color base08 */
+	"#33ff00", /* color base0B */
+	"#ff0099", /* color base0A */
+	"#0066ff", /* color base0D */
+	"#cc00ff", /* color base0E */
+	"#00ffff", /* color base0C */
+	"#ffffff", /* color base07 */
+
+	/* additional colors */
+	"#ff9900", /* color base09 */
+	"#3300ff", /* color base0F */
+	"#404040", /* color base01 */
+	"#606060", /* color base02 */
+	"#c0c0c0", /* color base04 */
+	"#e0e0e0", /* color base06 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#606060", /* color base02 */
+	"#ffffff", /* color base07 */
 };
 
 
