diff --git a/config.def.h b/config.def.h
index fd09d72..d084e0a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,30 +70,38 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#343d46", /* color base00 */
+	"#cc6666", /* color base08 */
+	"#b5bd68", /* color base0B */
+	"#f0c674", /* color base0A */
+	"#81a2be", /* color base0D */
+	"#b294bb", /* color base0E */
+	"#8abeb7", /* color base0C */
+	"#dfe1e8", /* color base05 */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#a7adba", /* color base03 */
+	"#cc6666", /* color base08 */
+	"#b5bd68", /* color base0B */
+	"#f0c674", /* color base0A */
+	"#81a2be", /* color base0D */
+	"#b294bb", /* color base0E */
+	"#8abeb7", /* color base0C */
+	"#ffffff", /* color base07 */
+
+	/* additional colors */
+	"#de935f", /* color base09 */
+	"#a3685a", /* color base0F */
+	"#4f5b66", /* color base01 */
+	"#65737e", /* color base02 */
+	"#c0c5ce", /* color base04 */
+	"#eff1f5", /* color base06 */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#65737e", /* color base02 */
+	"#ffffff", /* color base07 */
 };
 
 
